packages/cli/src/cmds/preview/preview.ts

```diff
@@ -180,6 +186,8 @@ export default class Preview extends YargsCommand {
       return err(errors.RequiredPathNotExists(botRoot));
     }
 
+    const [funcToolChecker, dotnetChecker] = await this.handleDependences(includeBackend);
```

Comment:
Error handleing for `this.handleDependences`.

Reply:
has already handled

Reply:
Compile error here.